IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+the design before the final size and interface details must be provided to LI. After this point, there will be more internal
+design and development for the remainder of the semester with a goal of purchasing all materials by the semester's end.
+If this timeline can be adhered to, then the entire spring semester and summer months leading up to the competition can be
+used for manufacturing, programming, integration, testing, and characterization.
+
+\subsection{Deliverables}
+\label{subsec:Deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+In order to be compliant with the SDL Payload Challenge requirements, a poster is needed as part of the judging criteria.
+Other deliverables for this project include the open-source technical documentation such as circuit schematics, CAD drawings,
+and embedded software. The final deliverable is a functioning payload module conforming to the IREC SDL payload challenge
+standard with accompanying ground station hardware and operating documentation.
+
+\section{Externalities}
+\label{subsec:Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+\label{subsec:Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+To participate in this endeavor, there are no prerequisite skills. However, for the project to succeed, there will need to be
+individuals who are proficient in computer-aided design and mechanical analysis for the structure of the payload. These members
+are responsible for the selection of materials and the construction of the frame, airbag deployment, and the return parachute.
+There will also need to be individuals proficient in electronic design, specifically with respect to printed circuit boards.
+These members are responsible for sensor selection and implementation, power regulation, and maintaining electrical connections
+between each part of the payload. Individuals proficient in programming will be required for embedded systems design, ground-station processing,
+and the overarching communication network that is needed to meet the goals of the project.  Finally, there will need to be some individuals
+who are willing to fill the role of system engineers who will be responsible for communicating with the separate LI team and acting as a resource
+to those designing the hardware and software.
+
+\subsection{Funding Requirements}
+\label{subsec:Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Although the preliminary design has not yet been completed, component selection has already begun and estimates for total cost have been
```

Comment:
PDD should be independent of the work that has taken place already. remember, this is a definition document not a report of what happened but of what will/could be done. independent of time!